src/activate.ts

```diff
@@ -0,0 +1,30 @@
+require('../config/configModule')
```

Comment:
Wouldn't it be better to have `configModule` export the config values rather than have it populate `process.env` as a side-effect?

Nit: I think `config.js` would be a simpler name.